### ISSUE TITLE
[Bug] Auto-detection doesnt work (Nim docs)

### ISSUE THREAD

**Issue by ShalokShalom** [darkreader/darkreader#8595]
# Bug Report

## Website Address

https://nim-lang.org/docs/intern.html

## Bug Description

Darkreader detects in its new auto-detection mode the dark mode of the website correctly. It still applies its theme. 

## Steps To Reproduce

1.) Go to the site
2.) Activate the new design and auto-detect

## Expected Behavior

Theme stays the same

## Actual Behavior

DarkReader activates a theme.  

## Screenshots

![Screenshot_201](https://user-images.githubusercontent.com/6344099/161635144-8c6b7c64-0e29-4b27-af6e-4e909ba21324.png)
![Screenshot_198](https://user-images.githubusercontent.com/6344099/161635147-abbc76fa-571d-4660-8521-a78e7b93c333.png)

## System Information

- OS:  Garuda
- Browser:  Firedragon 98
- Dark Reader Version:  current

**Comment by Gusted** [comment]
CC @alexanderby

**Comment by somebody1234** [comment]
Seems like it might try detecting too early on some websites? Might be worth running detection again after all css has been loaded, if that's even possible. otherwise after 1 second, or every 100ms for the first second, is better than nothing

**Comment by ShalokShalom** [comment]
Yeah, I had this already couple of times, that DarkReader applied its magic a little bit too early.

**Comment by antwan** [comment]
This is the same on many websites with auto-theme switch (github, linkedin, etc).
